packages/model-fragments/lib/fragments/attributes.js

```diff
@@ -319,13 +319,15 @@ function getDefaultValue(record, options, type) {
 
   if (typeof options.defaultValue === "function") {
     value = options.defaultValue();
-  } else if (options.defaultValue) {
+  } else if ("defaultValue" in options) {
```

Comment:
I'm not a huge fan of `in`, can you change this to `options.defaultValue !== undefined`.